modules/space/translation/spicetranslation.cpp

```diff
@@ -159,11 +178,20 @@ glm::dvec3 SpiceTranslation::position(const UpdateData& data) const {
     if (_fixedEphemerisTime.has_value()) {
         time = *_fixedEphemerisTime;
     }
+
+    SpiceManager::AberrationCorrection corr = {};
```

Comment:
Why does the translation and rotation implement the light travel time correction differently? Here it seems to use SPICE built in functionality but in the rotation it seems to be a more manual process where the time is shifted outside of SPICE.